doc/getting-started.md

```diff
@@ -23,17 +23,30 @@ The configuration must be passed to Ignition through the designated data source.
 
 ## Troubleshooting
 
+### Gathering Logs
+
 The single most useful piece of information needed when troubleshooting is the log from Ignition. Ignition runs in multiple stages so it's easiest to filter by the syslog identifier: `ignition`. When using systemd, this can be accomplished with the following command:
 
 ```
-journalctl --identifier=ignition
+journalctl --identifier=ignition --all
 ```
 
 In the event that this doesn't yield any results, running as root may help. There are circumstances where the journal isn't owned by the systemd-journal group or the current user is not a part of that group.
 
-In the vast majority of cases, it will be immediately obvious why Ignition failed. If it's not, inspect the config that Ignition wrote into the log. This shows how Ignition interpreted the supplied configuration. The user-provided config may have a misspelled section or maybe an incorrect hierarchy.
+### Validating the Configuration
+
+One common cause for Ignition failures is a malformed configuration (e.g. a misspelled section or incorrect hierarchy). Ignition will log errors, warnings, and other notes about the configuration that it parsed, so this can be used to debug issues with the configuration provided. As a convenience, CoreOS hosts an [online validator][validator] which can be used to quickly verify configurations.
+
+### Enabling Services
+
+When Ignition enables services, it doesn't directly create the symlinks necessary; it leverages [systemd presets][preset]. Presets are only evaluated on first-boot, which can result in confusion if Ignition is forced to run more than once. Any services which are enabled in the configuration after the first boot won't actually be enabled by default. `systemctl preset-all` will need to be manually invoked to enable the services.
+
+Running Ignition multiple times is not a recommended or typical use-case, so it's unlikely that this situation will arise.
```

Comment:
"Running multiple times is not recommended" -> "Running after first boot is not recommended"
Sometimes this comes up in the way we use Ignition on first PXE boot to install CoreOS (provision disk with CoreOS) and then use Ignition on first boot from disk (provision disk so machine assumes a role).